recipes/gnuplot/4.6.0/meta.yaml

```diff
@@ -1,25 +1,36 @@
 package:
   name: gnuplot
   version: "4.6.0"
+
 build:
-  number: 1
+  number: 2
+
 source:
-  git_url: https://github.com/gnuplot/gnuplot.git
-  git_rev: Release_4_6_0
+  url: https://github.com/gnuplot/gnuplot/archive/Release_4_6_0.tar.gz
+  sha256: c0885a5b4f5c1f64545b9852c0184ec5fd230df4c7c99e5a3cbd25dd441e5206
   patches:
     - fix-automake.patch
     - fix-proto-int.patch
+
 requirements:
   build:
-    - autoconf
-    - m4
+    - gcc [not osx]
+    - libgcc [not osx]
+    - llvm [osx]
+    - pkg-config [osx]
+    - libgd *.bioconda
```

Comment:
IIRC the problem is that gnuplot relied (relies?) on the deprecated `gdlib-config` script, which is not installed with libgd from defaults.

All versions of libgd in conda-forge should work fine. For compatibility I would pin to `libgd>2.1`